src/blocks/form/mailerlite.js

```diff
@@ -207,13 +206,17 @@ class MailerLiteControls extends Component {
 						{ ! isFetching && hasGroup && (
 							<Fragment>
 								<h2 className="kt-heading-size-title">{ __( 'Select Group', 'kadence-blocks' ) }</h2>
-								<SelectControl
+								<div className="mailerlite-select-form-row">
+								<Select
```

Comment:
Because react select saves as an array holding both the value and the label we will want to change this for backward compatibility. Also if they change their group name in mailerlite it could break this as well, so instead, we need to specify to only save the value which is the id of the group.

Reply:
That makes sense. I committed those changes, updated the build files, tested with an existing list selection (saved from the SelectControl), and tested with a new list selection